addons/account/models/ir_actions_report.py

```diff
@@ -57,7 +57,7 @@ def _retrieve_stream_from_attachment(self, attachment):
             record = self.env[attachment.res_model].browse(attachment.res_id)
             try:
                 return pdf.add_banner(stream, record.name, logo=True)
-            except (ValueError, PdfStreamError):
+            except (ValueError, PdfStreamError, PdfReadError, TypeError):
```

Comment:
is TypeError necessary?

Reply:
@nle-odoo yep you can get it with odoo/addons/base/tests/minimal.pdf